osx/post_installation.sh

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
+
+# Packages postscript for KA-Lite
+#
+# Steps
+# 1. Symlink kalite executable  to /usr/local/bin.
+# 2. Export KALITE_PYTHON env that point to Pyrun directory.
+# 3. Create plist in ~/Library/LaunchAgents folders.
+# 4. Run shebangcheck that check the BIN_PATH that points to the python/pyrun interpreter to use.
+# 5. Run kalite manage syncdb --noinput.
+# 6. Run kalite manage init_content_items --overwrite.
+# 7. Run kalite manage setup --noinput.
+# 8. Run kalite manage unpack_assessment_zip <assessment_path>.
+# 9. Change the owner of a $HOME/.kalite.
+
```

Comment:
Also mention that we use `/Users/Shared/ka-lite` as the installation location of the `ka-lite` source (and Pyrun).